src/core/thread/network_data_local.cpp

```diff
@@ -58,9 +58,13 @@ Local::Local(ThreadNetif &aThreadNetif):
 ThreadError Local::AddOnMeshPrefix(const uint8_t *aPrefix, uint8_t aPrefixLength, int8_t aPrf,
                                    uint8_t aFlags, bool aStable)
 {
+    ThreadError error = kThreadError_None;
     PrefixTlv *prefixTlv;
     BorderRouterTlv *brTlv;
 
+    VerifyOrExit(memcmp(aPrefix, mNetif.GetMle().GetMeshLocalPrefix(), aPrefixLength / 8),
```

Comment:
Both aPrefix and mNetif.GetMle().GetMeshLocalPrefix() is of type `const uint8_t*`, not `Address`. Wrap the two into `Address` objects?